kernel/kernel-impl/src/main/sql/oracle/sakai_config_item.sql

```diff
@@ -27,12 +27,4 @@ ALTER TABLE SAKAI_CONFIG_ITEM
 CREATE INDEX SCI_NODE_IDX ON SAKAI_CONFIG_ITEM (NODE ASC);
 CREATE INDEX SCI_NAME_IDX ON SAKAI_CONFIG_ITEM (NAME ASC);
 
-CREATE SEQUENCE SAKAI_CFG_ITEM_SEQ;
-
-CREATE OR REPLACE TRIGGER SCI_ID_AI
-BEFORE INSERT ON SAKAI_CONFIG_ITEM
-FOR EACH ROW
-BEGIN
-  SELECT SAKAI_CFG_ITEM_SEQ.NEXTVAL INTO :new.ID FROM dual;
-END;
-/
+CREATE SEQUENCE SAKAI_CONFIG_ITEM_S;
```

Comment:
I don't know why, the code is not using sql statements to insert elements in the database, delegates the work in Hibernate:

https://github.com/sakaiproject/sakai/blob/master/kernel/kernel-impl/src/main/java/org/sakaiproject/config/impl/HibernateConfigItemDaoImpl.java

And the hbm definition allows hibernate to use the sequence and assign the right value to ID field. I've tested this without any trigger and it works, the table is populated properly with all the sakai properties. Sorry if I miss something.